tests/common-lib/clusterAccess.js

```diff
@@ -12,7 +12,9 @@ const clusterLogin = () => {
     execSync(
       `oc login -u ${config.get('options:hub:user')} -p ${config.get(
         'options:hub:password'
-      )} --server=https://api.${config.get('options:hub:baseDomain')}:6443`
+      )} --server=https://api.${config.get(
+        'options:hub:baseDomain'
+      )}:6443 --insecure-skip-tls-verify`
```

Comment:
Is it okay too use insecure-skip-tls-verify in this environment? Might want to double check

Reply:
For this test, it is okay to use. We are using this also within the `start-tests.sh` script.